XenAdmin/Dialogs/OptionsPages/UpdatesOptionsPage.cs

```diff
@@ -85,6 +88,22 @@ public void Save()
                 Properties.Settings.Default.AllowPatchesUpdates = AllowXenServerPatchesCheckBox.Checked;
             if (AllowXenServerUpdatesCheckBox.Checked != Properties.Settings.Default.AllowXenServerUpdates)
                 Properties.Settings.Default.AllowXenServerUpdates = AllowXenServerUpdatesCheckBox.Checked;
+
+            if(refreshUpdatesTab)
+            {
+                Updates.CheckForUpdates(false);
+            }
+        }
+
+        /// <summary>
+        /// Returns true if at least one other box has been checked in Updates Options. Otherwise returns false.
+        /// </summary>
+        /// <returns></returns>
+        private bool AtLeastOneOptionHasBenChecked()

```

Comment:
it looks good, but I would rename this, because the naming is misleading. I think what it does is more like IsCheckForUpdateRequired()